<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>熄灯游戏</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>150</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HarmonyOS Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>经典3*3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>260</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HarmonyOS Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>挑战5*5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_x">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>480</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HarmonyOS Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>探索模式</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>370</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HarmonyOS Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>自定义大小</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>181</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;规则:&lt;/p&gt;&lt;p&gt;点击灯泡可以改变灯泡状态&lt;/p&gt;&lt;p&gt;但是相邻的灯泡也会同时改变状态&lt;/p&gt;&lt;p&gt;熄灭所有灯泡即为获胜&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
